Makefile

```diff
@@ -63,7 +63,8 @@ endif
 
 # Allow overriding manifest generation destination directory
 MANIFEST_ROOT ?= ./config
-CRD_ROOT ?= $(MANIFEST_ROOT)/crd/bases
+CRD_ROOT ?= $(MANIFEST_ROOT)/default/crd/bases
+SUPERVISOR_CRD_ROOT ?= $(MANIFEST_ROOT)/supervisor/crd
```

Comment:
CRDs split into two different locations: default and supervisor. These end up being the primary difference between `infrastructure-components` and `infrastructure-components-supervisor`